README.md

```diff
@@ -29,15 +33,15 @@ unit_circle_bz = BezierCurve(
 
 # A Real-valued function
 f(x, y, z) = abs(x + y)
-f(p) = f(p.coords...)
+f(p) = f(p.coords.x, p.coords.y, p.coords.z)
```

Comment:
Btw, is there a more concise way for this? When I tested `p.coords...` locally, it didn't work anymore. This is maybe also a question for @juliohm.

Reply:
We have two distinct concepts in Meshes.jl now:

1. The `coords` of a point is a complicated struct with important information to locate it in the physical world. It is not a `AbstractVector` nor an iterable. We rarely use `coords` directly in Euclidean geometry.
2. The `to` function is preferred here in the sense that it returns the `Vec` from the origin to the point. It will be a `StaticVector` with support for LinearAlgebra.

I would write the function as

```julia
f(p) = f(to(p)...)
```

Reply:
Makes sense. Thanks! See [8c5d0a7](https://github.com/mikeingold/MeshIntegrals.jl/pull/43/commits/8c5d0a72fb8ab383b626baffa704a2c8698bdf83).